detekt.yml

```diff
@@ -84,7 +84,7 @@ formatting:
     active: true
     autoCorrect: true
   ExpressionBodySyntaxLineBreaks:
-      active: true
+      active: false
```

Comment:
Ok. I'll do what we agreed on in slack for this PR:
* Auto-fixed ExpressionBodySyntaxLineBreaks violations with `./gradlew detekt` (running it multiple times).
* When I just had a few `MaxLineLength` violations I regenerated baseline with `./gradlew detektEstablishAcceptedErrors`.